### DISCUSSION TITLE
如何在全屏的时候也能显示窗口化的这种小进度条

### DISCUSSION THREAD

**Discussion by MeetDead** [hooke007/mpv_PlayKit#298] [mpv 通用]
![屏幕截图 2023-04-14 190515](https://user-images.githubusercontent.com/87464979/232028587-02e705a5-d952-4dac-b4ed-0c7e1455ca0b.png)

**Comment by zhongfly** [comment]
请学会阅读配置文件
https://github.com/hooke007/MPV_lazy/blob/a8ce1dc306dfa21b398f768b3848a54d269d3881/portable_config/script-opts/uosc.conf#L15-L16

连中文注释都有，有什么理由不自己看呢
